src/schema.js

```diff
@@ -192,6 +192,18 @@ const queryArticleTagsSchema = Joi.object({
 	}
 })
 
+const inviteUserSchema = Joi.object({
+	body: {
+		email: Joi.string()
+			.email({
```

Comment:
lets use just `email()`, the existing ones there are not ideal and they are things we want to change